plugins/chatsizediff.lua

```diff
@@ -0,0 +1,67 @@
+PLUGIN.name = "Chat Size Difference"
+PLUGIN.desc = "Overrides size for different chat types."
+PLUGIN.author = "Zoephix"
+
+nut.config.add("chatSizeDiff", true, "Whether or not to use different chat sizes.", nil, {
```

Comment:
I'd set that to false by default, so that the current look of the chatbox isn't affected and the option would need to be manually enabled for the effects to take place